### PR TITLE
Mp wandb

### PR DESCRIPTION
**Pull request by maciek-pioro**
# Description
todo

# Checklist
   - [x] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [x] Are all variables set with logical defaults that are backward compatible?
   - [x] I attached link to neptune if it was necessary
      - I did not. Wandb tests pass and @crewtool checked it out.
   - [x] Have you successfully executed the linter and tests?
   - [x] Are all functions concise and don't require splitting?
   - [x] Is there documentation for arguments and complex logic?
   - [x] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [x] Are all functions placed in appropriate files?
   - [x] Is the PR name meaningful and descriptive?
   - [x] Is PR description provided, or unnecessary?
  
Other questions:
   - [x] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [x] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [x] If there is a need for a second reviewer I requested additional review 

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

File: lizrd/support/logging.py [hunk 1]
```diff
@@ -251,6 +252,77 @@ def flush_if_necessary(self):
...
+        *,
+        title: str,
+        value: float,
+        iteration: int,
+        series: Optional[str] = None,
+    ):
+        path = self._make_path(title, series)
+        assert (not math.isnan(value)) and (
+            not math.isinf(value)
+        ), f"Trying to log {path} as {value}. Neptune doesn't allow logging NaN or Inf."
```
Comment (crewtool) [inline lizrd/support/logging.py:291]: Wandb

File: research/conditional/train/configs/test/test_baseline_wandb.yaml
Comment (crewtool) [inline research/conditional/train/configs/test/test_baseline_wandb.yaml:1]: Lets move it to `configs` in root

**Review by crewtool — APPROVED** [review]
LGTM! It worked for me

File: lizrd/support/logging.py (hunk 1 above)
Comment (maciek-pioro) [reply lizrd/support/logging.py]: Now we log inf / nan (Wandb allows that)

File: research/conditional/train/configs/test/test_baseline_wandb.yaml
Comment (maciek-pioro) [reply research/conditional/train/configs/test/test_baseline_wandb.yaml:1]: moved

**Pull request merged**